### PR TITLE
DRV-85: query timeout header

### PR DESCRIPTION
**Pull request by evbruno**

### CODE REVIEW

File: FaunaDB.Client.Test/TestCase.cs [hunk 1]
```diff
@@ -98,4 +97,19 @@ public MockClientIO(RequestResult resp)
         public Task<RequestResult> DoRequest(HttpMethodKind method, string path, string data, IReadOnlyDictionary<string, string> query = null) =>
             Task.FromResult(resp);
     }
+
+    class HttpClientWrapper : HttpClient
+    {
+
+        HttpRequestMessage LastMessage;
```
Comment (marrony) [inline FaunaDB.Client.Test/TestCase.cs:104]:
```csharp
public HttpRequestMessage LastMessage { get; private set; }
```
if I not mistake

File: FaunaDB.Client.Test/TestCase.cs [hunk 2]
```diff
@@ -40,21 +42,18 @@ async Task SetUpAsync()
...
             var port = Env("FAUNA_PORT", "8443");
-            var secret = Env("FAUNA_ROOT_KEY", "secret");
-            var endpoint = $"{scheme}://{domain}:{port}";
-            var httpClient = new HttpClient();
-            httpClient.BaseAddress = new Uri("https://www.wrongbaseaddress.com");
 
-            rootClient = new FaunaClient(secret: secret, endpoint: endpoint, httpClient: httpClient);
+            faunaSecret = Env("FAUNA_ROOT_KEY", "secret");
+            faunaEndpoint = $"{scheme}://{domain}:{port}";
+            rootClient = new FaunaClient(secret: faunaSecret, endpoint: faunaEndpoint, httpClient: new HttpClient());
```
Comment (marrony) [inline FaunaDB.Client.Test/TestCase.cs:48]: aren't the driver already instantiating `HttpClient` if we do not provide one? if so, you can just remove this argument

Comment (evbruno) [reply FaunaDB.Client.Test/TestCase.cs]: fixed

**Comment by evbruno** [comment]
updated the `query` method, README file... good for further reviewing

**Pull request merged**